Makefile

```diff
@@ -100,7 +104,9 @@ create-install-files-dev:
 	INSTALL_FILES_FOLDER=../../installfilesdev \
 	IMG=$(NS)$(IMAGE_NAME_OPERATOR):${IMAGE_TAG} \
 	IMAGE_NAME_INIT_CONTAINER=$(NS)$(IMAGE_NAME_INIT_CONTAINER) \
+	IMAGE_NAME_INIT_CONTAINER_WIN=$(NS)$(IMAGE_NAME_INIT_CONTAINER_WIN) \
 	IMAGE_NAME_NODE_AGENT=$(NS)$(IMAGE_NAME_NODE_AGENT) \
+	IMAGE_NAME_NODE_AGENT_WIN=$(NS)$(IMAGE_NAME_NODE_AGENT_WIN) \

```

Comment:
overall comment: I'm pretty sure that docs/README mention that Thundernetes is only used for Linux game servers, can we take a quick look to fix this? i.e. where we say "Linux game servers" just mention "game servers in both Windows and Linux" or something?